### PR TITLE
UICONSET-161: Add Authorization Policies to the consortium manager

### PR DESCRIPTION
**Pull request by alisher-epam**
## Purpose

[UICONSET-161](https://folio-org.atlassian.net/browse/UICONSET-161): Add Authorization Policies to the consortium manager
We have a supporting PR [https://github.com/folio-org/ui-consortia-settings/pull/137](https://github.com/folio-org/ui-consortia-settings/pull/137)
## Approach

https://github.com/user-attachments/assets/39570e7a-5754-423a-9151-19f726ce6339

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/routes/ConsortiumManager/settings/authorization-policies/AuthorizationPoliciesSettings.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { useMemo } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import AuthorizationPolicies from '@folio/authorization-policies';
```
Comment (NikitaSedyx) [inline src/routes/ConsortiumManager/settings/authorization-policies/AuthorizationPoliciesSettings.js:4]: don't import module in another module, for this purpose `stripes-authorization-components` was created

**Review by NikitaSedyx — CHANGES_REQUESTED** [review]
no modules allowed as dependencies

**Comment by alisher-epam** [comment]
@NikitaSedyx can you review it please?

File: src/routes/ConsortiumManager/settings/authorization-policies/constanta.js
Comment (usavkov-epam) [inline src/routes/ConsortiumManager/settings/authorization-policies/constanta.js:1]: `constants.js`

File: src/routes/ConsortiumManager/settings/authorization-policies/constanta.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+};
+
+export const VISIBLE_COLUMNS = [
+  COLUMN_NAMES.name,
+  COLUMN_NAMES.description,
+  COLUMN_NAMES.updated,
+  COLUMN_NAMES.updatedBy,
+];
+
+export const getResultsFormatter = ({ users }) => ({
```
Comment (usavkov-epam) [inline src/routes/ConsortiumManager/settings/authorization-policies/constanta.js:27]: I don't think that function should be stored in the file with the name "constants"

Comment (alisher-epam) [reply src/routes/ConsortiumManager/settings/authorization-policies/constanta.js]: updated

File: src/routes/ConsortiumManager/settings/authorization-policies/constanta.js
Comment (alisher-epam) [reply src/routes/ConsortiumManager/settings/authorization-policies/constanta.js:1]: updated

**Review by usavkov-epam — APPROVED** [review]

**Review by UladzislauKutarkin — APPROVED** [review]

**Pull request merged**